plugins/lookup/random_string.py

```diff
@@ -16,6 +16,7 @@
     version_added: '3.2.0'
     description:
       - Generates random string based upon the given constraints.
+      - Uses L(C(os.urandom),https://docs.python.org/3/library/os.html#os.urandom), so is strong enough for cryptographic purposes.
```

Comment:
I would reference to what this plugin actually uses (`random.SystemRandom`). That `random.SystemRandom` is using `os.urandom` is something we cannot guarantee, but which depends on the Python environment used to run ansible-core.
```suggestion
      - Uses L(random.SystemRandom,https://docs.python.org/3/library/random.html#random.SystemRandom), so should be strong enough for cryptographic purposes.
```